hw/top_earlgrey/fpv/top_earlgrey_dedicated_fpv_cfgs.hjson

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the master cfg hjson for FPV. It imports ALL individual FPV
+  // dedicated cfgs of the IPs. This enables to run them all as a regression in one shot.
+  name: top_earlgrey_dedicated_ip_batch_fpv
+
+  import_cfgs: [// common server configuration for results upload
+                "{proj_root}/hw/data/common_project_cfg.hjson"
+                "{proj_root}/hw/formal/data/dedicated_fpv_cfgs.hjson"]
```

Comment:
A master cfg list should ideally not be imported in other cfgs. It needs to be the entry point to the tool as input. What is the intent of organizing it this way?

Reply:
I see, I will go back the read the `testplan` readme. Here I wanted to distinguish two sets of FPV testbenches:
1. `{proj_root}/hw/data/common_project_cfg.hjson` includes DUTs that partially verified by FPV: only includes bind file with TLUL assertions and internal assertions. (There is no need to collect coverage on these modules)
2. `{proj_root}/hw/formal/data/dedicated_fpv_cfgs.hjson` includes DUTs that fully verified by FPV. (They need to collect coverage)
3. `hw/top_earlgrey/fpv/top_earlgrey_dedicated_fpv_cfgs.hjson` runs both 1) and 2) modules as a weekly regression results.

Reply:
It is somehow working and that is good to see. Let's leave it at that and fix any tool issues we might run into later.

Reply:
Sounds good, If happened, I will work on the fix related to this.